homeassistant/components/ads/switch.py

```diff
@@ -1,30 +1,25 @@
 """Support for ADS switch platform."""
-import voluptuous as vol
-
-from homeassistant.components.switch import PLATFORM_SCHEMA, SwitchEntity
-from homeassistant.const import CONF_NAME
-import homeassistant.helpers.config_validation as cv
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.const import CONF_NAME, CONF_SWITCHES
 
 from . import CONF_ADS_VAR, DATA_ADS, STATE_KEY_STATE, AdsEntity
 
-DEFAULT_NAME = "ADS Switch"
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_ADS_VAR): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-    }
-)
-
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up switch platform for ADS."""
+    entities = []
+
+    if discovery_info is None:  # pragma: no cover
```

Comment:
Why `# pragma: no cover` when there are not tests currently?
```suggestion
    if discovery_info is None:
```